flow/envs/multiagent/traffic_light_grid.py

```diff
@@ -194,7 +192,7 @@ def get_state(self):
             observation = np.array(np.concatenate(
                 [speeds[rl_id_num], dist_to_intersec[rl_id_num],
                  edge_number[rl_id_num], density[local_edge_numbers],
-                 velocity_avg[local_edge_numbers], last_change[local_id_nums],
```

Comment:
do we want to remove this? it wasn't mentioned in the PR description. (making sure we aren't deleting something we want to keep)

Reply:
Yes @AboudyKreidieh. I emailed Cathy Wu few days back and discussed with her that we can remove this, as it is causing some bug (causing observation space not to be between 0 and 1)